CriteoPublisherSdk/Sources/Native/CRNativeLoader.m

```diff
@@ -41,6 +41,13 @@ @implementation CRNativeLoader
 
 #pragma mark - Life cycle
 
+- (instancetype)init {
```

Comment:
Using this init does not initialize `self.criteo` or `self.urlOpener`. Is it intended ?
Generally when overloading constructors, constructor will less parameters calls the one with more parameters with default value or nil (nil here). So the object is always initialized in the same manner.

Reply:
Done.